exporter/awsemfexporter/grouped_metric.go

```diff
@@ -83,6 +97,135 @@ func addToGroupedMetric(pmd *pdata.Metric, groupedMetrics map[interface{}]*Group
 			}
 		}
 	}
+
+	return nil
+}
+
+type kubernetesObj struct {
+	ContainerName string               `json:"container_name"`
+	Docker        internalDockerObj    `json:"docker"`
+	Host          string               `json:"host"`
+	Labels        internalLabelsObj    `json:"labels"`
+	NamespaceName string               `json:"namespace_name"`
+	PodID         string               `json:"pod_id"`
+	PodName       string               `json:"pod_name"`
+	PodOwners     internalPodOwnersObj `json:"pod_owners"`
+	ServiceName   string               `json:"service_name"`
+}
+
+type internalDockerObj struct {
+	ContainerID string `json:"container_id"`
+}
+
+type internalLabelsObj struct {
+	App             string `json:"app"`
+	PodTemplateHash string `json:"pod-template-hash"`
+}
+
+type internalPodOwnersObj struct {
+	OwnerKind string `json:"owner_kind"`
+	OwnerName string `json:"owner_name"`
+}
+
+func addKubernetesWrapper(labels map[string]string) error {
+	//create schema
+	schema := kubernetesObj{}
+	schema.ContainerName = "container"
+	schema.Docker =
+		internalDockerObj{
+			ContainerID: "container_id",
+		}
+	schema.Host = "NodeName"
+	schema.Labels =
+		internalLabelsObj{
+			App:             "app",
+			PodTemplateHash: "pod-template-hash",
+		}
+	schema.NamespaceName = "Namespace"
+	schema.PodID = "PodId"
+	schema.PodName = "PodName"
+	schema.PodOwners =
+		internalPodOwnersObj{
+			OwnerKind: "owner_kind",
+			OwnerName: "owner_name",
+		}
+	schema.ServiceName = "Service"
```

Comment:
```suggestion
	schema := kubernetesObj{
		ContainerName: "container",
		Docker: internalDockerObj{
			ContainerID: "container_id",
		},
		Host: "NodeName",
		Labels: internalLabelsObj{
			App:             "app",
			PodTemplateHash: "pod-template-hash",
		},
		NamespaceName: "Namespace",
		PodID: "PodId",
		PodName: "PodName",
		PodOwners: internalPodOwnersObj{
			OwnerKind: "owner_kind",
			OwnerName: "owner_name",
		},
		ServiceName: "Service",
	}
```

It is idiomatic to use the struct literal initialization form.